spec/lib/sidekiq/form526_backup_submission_process/processor_spec.rb

```diff
@@ -26,4 +26,26 @@
       end
     end
   end
+
+  describe '#choose_provider' do
+    let(:account) { create(:account) }
+    let(:submission) { create(:form526_submission, user_uuid: account.idme_uuid) }
+
+    it 'delegates to the ApiProviderFactory with the correct data' do
+      auth_headers = {}
+      expect(ApiProviderFactory).to receive(:call).with(
+        {
+          type: ApiProviderFactory::FACTORIES[:generate_pdf],
+          provider: nil,
+          options: { auth_headers:, breakered: true, icn: account.icn },
+          current_user: OpenStruct.new({ flipper_id: submission.user_uuid }),
+          feature_toggle: ApiProviderFactory::FEATURE_TOGGLE_GENERATE_PDF
+        }
+      )
+
+      subject
+        .new(submission.id, get_upload_location_on_instantiation: false)
+        .choose_provider(auth_headers)
```

Comment:
Default provider is no longer defined as `nil`, so I think you need to include a `provider` argument in this method call.